tests/TestCase/Controller/ControllerFactoryTest.php

```diff
@@ -888,14 +888,15 @@ public function testInvokePassedParamUnsupportedReflectionType(): void
                 'plugin' => null,
                 'controller' => 'Dependencies',
                 'action' => 'unsupportedTypedUnion',
```

Comment:
Should we rename this method to `typedUnion`?

Reply:
Done